test/integration/BUILD

```diff
@@ -642,6 +642,18 @@ envoy_cc_test(
     ],
 )
 
+envoy_cc_test(
```

Comment:
Please move this test to the extension directory and make it an extension only test.

Reply:
Done, thanks.